UKHomeOffice/sgmr-service
# 1.1.2 - SPCR-334 Make cancel voyage plan CTA a button

## Description
Currently the cancel voyage CTA is a link, styled as a button. This needs to be made into an actual button to improve accessibility.

## To test
- Turn on voice over
- Navigate to manage voyage plans
- Click on a draft report
- Tab through the form
- The screen reader should read out the cancel voyage plan CTA as a button
 not a link

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
